app/src/main/java/com/coveros/coverosmobileapp/PostList.java

```diff
@@ -0,0 +1,268 @@
+package com.coveros.coverosmobileapp;
+
+import android.app.ListActivity;
+import android.content.DialogInterface;
+import android.os.Bundle;
+
+// importing tools for WordPress integration
+
+import android.content.Intent;
+import android.support.v7.app.AlertDialog;
+import android.util.Log;
+import android.view.View;
+import android.widget.AbsListView;
+import android.widget.AdapterView;
+import android.widget.ListView;
+
+import com.android.volley.NetworkResponse;
+import com.android.volley.Request;
+import com.android.volley.RequestQueue;
+
+
+import com.android.volley.Response;
+import com.android.volley.VolleyError;
+import com.android.volley.toolbox.StringRequest;
+import com.android.volley.toolbox.Volley;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+/**
+ * Created by Maria Kim on 6/9/2017
+ * Creates ListView that displays list of titles of blog post_list.
+ */
+public class PostList extends ListActivity {
+
+    List<Post> posts = new ArrayList<>();
```

Comment:
Some of these could even be local variables - consider lowering their scope to be local to whatever method they're used